tests/terraform/graph/checks/resources/GCRContainerVulnerabilityScanningEnabled/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "google_project_services" "pass_1" {
+  project = "your-project-id"
+  services   = ["iam.googleapis.com", "cloudresourcemanager.googleapis.com", "containerscanning.googleapis.com"]
+}
+
+resource "google_project_service" "pass_2" {
+  project = "your-project-id"
+  service = "containerscanning.googleapis.com"
+
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+
+  disable_dependent_services = true
+}
```

Comment:
seeing the test, I think we should remove the resource type `google_project_service` to be checked, because it will flag every `google_project_service` resource, which configures a different api service.

Reply:
yes let me know is their anyway to check this resource also ? if not, I can remove the resource from the check

Reply:
i don't think it is possible without creating too many false positives. would be great if you can remove `google_project_service` from the original check

Reply:
removed the false positive resource, Thanks

Reply:
@gruebel updated PR as per review comments please review